src/form-elements/FormElementArcGISWebMap.tsx

```diff
@@ -66,79 +66,114 @@ function FormElementArcGISWebMap({
   >(
     (e, data) => {
       onChange(element, {
-        value: {
-          ...value,
+        value: (existingValue) => ({
+          ...existingValue,
           snapshotImages: (!data.value
             ? undefined
             : Array.isArray(data.value)
               ? data.value
               : data.value(
-                  value?.snapshotImages,
+                  existingValue?.snapshotImages,
                 )) as SubmissionTypes.FormSubmissionAttachment[],
-        },
+        }),
       })
     },
-    [element, onChange, value],
+    [element, onChange],
   )
 
+  // Have a empty function to set is dirty because there are multiple ways
+  // to provide input into this component and we don't want any of them
+  // to trigger the validation message(s) for the other means of input.
```

Comment:
```suggestion
  // An empty function to set is dirty because there are multiple ways
  // to provide input into this component and we don't want any of them
  // to trigger the validation message(s) for the other means of input.
```